app/services/google/scrape_service.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+module Google
+  class ScrapeService
+    attr_reader :keyword
+
+    def initialize(keyword_id:)
+      @keyword_id = keyword_id
+    end
+
+    # rubocop:disable Metrics/MethodLength
+    # rubocop:disable Metrics/AbcSize
+    def call
```

Comment:
as this method could raise the exception, let follow the Ruby convention 

- use `!` suffix if the method could raise exception

https://albertoalmagro.com/three-things-you-must-know-about-ruby-bang-methods/


```suggestion
    def call!
```

Reply:
Fixed in https://github.com/Goose97/google-scraper-ruby/pull/20/commits/1ef3fbf1560eb98c39c5780571d2df1442e949a2